install-dependecies.sh

```diff
@@ -0,0 +1,69 @@
+echo "Updating Package list to install latest depencies"
+#Update package lists to retrieve latest version of packages
+sudo apt-get update
+
+echo "Insatlling Python Index Package (pip)."
+echo "We will require pip to install some of the dependencies"
+#install Python Index Package
+sudo apt-get install python-pip
+sudo apt-get install python3-pip
+sudo pip install --upgrade pip
+
+echo "--------------------------------------------------------"
+echo "### Installing package dependencies for IRC LogParser ###"
+echo "--------------------------------------------------------"
+
+echo "Installing networkx"
+#Install newtworkx package
+sudo pip install networkx
+echo "Finished installing networkx"
+
+echo "--------------------------------------------------------"
+echo "Installing numpy"
+#Install numpy
+sudo apt-get install python-numpy
+echo "Finished installing numpy"
+
+echo "--------------------------------------------------------"
+echo "Installing Matplotlib"
+#Install Matplotlib
+sudo apt-get install python-matplotlib
+echo "Finished installing Matplotlib"
+
+echo "--------------------------------------------------------"
+echo "Installing pygraphviz"
+#Install pygraphviz
+git clone https://github.com/pygraphviz/pygraphviz.git
+cd pygraphviz
+sudo python ./setup.py install
+cd ..
+sudo rm -rf pygraphviz
+echo "Finished installing pygraphviz"
+
+echo "--------------------------------------------------------"
+echo "Installing scipy and scikit learn"
+#Install scipy and sklearn
+sudo apt-get install python-scipy
+pip install -U scikit-learn
+echo "Finished installing scipy and scikit learn"
+
+echo "--------------------------------------------------------"
+echo "Installing NLTK and NLTK-Data"
+echo "Please Wait!!. This may take some time "
+
+#Install Natural Language Toolkit
+sudo pip install -U nltk
+python ./nltk_data.py
```

Comment:
@kaushik-rohit it is not recommended to create a seperate py script for a petty task.
this could help : http://stackoverflow.com/questions/2043453/executing-python-multi-line-statements-in-the-one-line-command-line